zulip_bots/zulip_bots/bots/virtual_fs/test_virtual_fs.py

```diff
@@ -19,6 +20,25 @@ class TestVirtualFsBot(BotTestCase):
                 '```\n'
                 'Use commands like `@mention-bot help write` for more details on specific\ncommands.\n')
 
+    def verify_dialog(self, conversation):
```

Comment:
Is there a reason you can't use the default BotTestCase.verify_dialog method?

Reply:
I wanted to check the case when there are multiple recipients. 
A better way would be to make a BotTestCase.mock_request_message function that sends multiple recipients.